rt/rt/OPCD$.MLC

```diff
@@ -1,13 +1,54 @@
+OPCD$DOS  TITLE 'Validate opcode table'
 ***********************************************************************
 * Start create : 2024-05-31
 * 1st delivery :
 * Designer     : AF Kornelis
 * Programmer   : AF Kornelis
-* Purpose      : Standard routine to
-*                compare a z390 optable listing
-*                with the HLASM equivalent
+* Purpose      : For validating opcode tables. 
+*                For each optable option xxx we have
+*                a separate OPCS$xxx.OPT file and a OPCS$xxx.TF1
+*                reference file produced by HLASM
+*                Assembling this program using the appropriate .OPT 
+*                file produces a listing that includes a table of
+*                all mnemonics supported for the specified optable.
+*                Running the assembled program will compare the table
+*                of supported menmonics against the reference file.
 ***********************************************************************
 
+         YREGS ,                       * register equates
+
+***********************************************************************
+*
+* Program entry and linkage
+*
+***********************************************************************
+OPCD$DOS CSECT ,                       *
+*PCD$DOS AMODE 24                      * Not supported in DOS
+*PCD$DOS RMODE 24                      * Not supported in DOS
```

Comment:
Delete these 2 lines???

Reply:
These two statements do not work when assembling with optable(DOS).
I encapsulated them under an AIF that tests whether these mnemonics are defined.